codegenerator/cli/src/hbs_templating/contract_import_templates.rs

```diff
@@ -281,13 +281,13 @@ impl Event {
         }
         .to_string();
 
-        let second_part = match is_fuel {
-            true => format!("${{{event_var_name}.transaction.id}}"),
-            false => format!("${{{event_var_name}.block.number}}"),
+        let block_number_field = match is_fuel {
+            true => "height",
+            false => "humber",
```

Comment:
I'm pretty sure this should be number?
```suggestion
            false => "number",
```